set.mm

```diff
@@ -660665,10 +660950,6 @@ not even needed (it can be any class).  (Contributed by Glauco
       $.
   $}
 
-  $( Plus infinity is not a real number.  (Contributed by Glauco Siliprandi,
-     23-Oct-2021.) $)
-  pnfnre2 $p |- -. +oo e. RR $=
-    ( cpnf cr pnfnre neli ) ABCD $.
```

Comment:
This movement to main should be documented in the file changes-set.txt.

Reply:
Yes! I had prepared the line in `changes-set.txt`, but for some reason I did not include it in the PR.